### ISSUE TITLE
tests are broken with latest memcached version

### ISSUE THREAD

**Issue by simonpasquier** [prometheus/memcached_exporter#67]
While debugging failures for #65 I noticed that they were due to a change in the memcached stats rather than the move from Go 1.12 to 1.13...

To confirm, I've [modified](
https://github.com/prometheus/memcached_exporter/pull/66) the Circle CI configuration so it runs tests against v1.5.16 and latest. This fails for the latest memcached image (see https://circleci.com/gh/prometheus/memcached_exporter/264).

The test error message is `main_test.go:142: want metrics to include "memcached_slab_mem_requested_bytes{slab=\"5\"} 194", have:...`. It seems that mem_requested stats have moved from the slabs stats to the items stats:

```diff
*** stats.slabs.1.5.16  2019-09-13 13:57:35.313452365 +0200
--- stats.slabs.1.5.17  2019-09-13 13:58:21.661464012 +0200
*************** STAT 1:total_chunks 10922
*** 5,11 ****
  STAT 1:used_chunks 1
  STAT 1:free_chunks 10921
  STAT 1:free_chunks_end 0
- STAT 1:mem_requested 68
  STAT 1:get_hits 2
  STAT 1:cmd_set 3
  STAT 1:delete_hits 0
--- 5,10 ----
*************** STAT 5:total_chunks 4369
*** 21,27 ****
  STAT 5:used_chunks 1
  STAT 5:free_chunks 4368
  STAT 5:free_chunks_end 0
- STAT 5:mem_requested 194
  STAT 5:get_hits 0
  STAT 5:cmd_set 1
  STAT 5:delete_hits 0
--- 20,25 ----
``` 

```diff
*** stats.items.1.5.16	2019-09-13 13:57:53.279456885 +0200
--- stats.items.1.5.17	2019-09-13 13:58:14.033462095 +0200
*************** STAT items:1:number_warm 0
*** 4,10 ****
  STAT items:1:number_cold 1
  STAT items:1:age_hot 0
  STAT items:1:age_warm 0
! STAT items:1:age 188
  STAT items:1:evicted 0
  STAT items:1:evicted_nonzero 0
  STAT items:1:evicted_time 0
--- 4,11 ----
  STAT items:1:number_cold 1
  STAT items:1:age_hot 0
  STAT items:1:age_warm 0
! STAT items:1:age 7
! STAT items:1:mem_requested 68
  STAT items:1:evicted 0
  STAT items:1:evicted_nonzero 0
  STAT items:1:evicted_time 0
*************** STAT items:5:number_warm 0
*** 31,37 ****
  STAT items:5:number_cold 1
  STAT items:5:age_hot 0
  STAT items:5:age_warm 0
! STAT items:5:age 188
  STAT items:5:evicted 0
  STAT items:5:evicted_nonzero 0
  STAT items:5:evicted_time 0
--- 32,39 ----
  STAT items:5:number_cold 1
  STAT items:5:age_hot 0
  STAT items:5:age_warm 0
! STAT items:5:age 7
! STAT items:5:mem_requested 194
  STAT items:5:evicted 0
  STAT items:5:evicted_nonzero 0
  STAT items:5:evicted_time 0
```

There might other differences though...

**Comment by simonpasquier** [comment]
From memcached 1.5.17 [changelog](https://github.com/memcached/memcached/wiki/ReleaseNotes1517):

> move mem_requested from slabs.c to items.c (internal cleanup)

**Comment by SuperQ** [comment]
Fixed with #70
